dash-pipeline/bmv2/defines.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _DASH_DEFINES_H_
+#define _DASH_DEFINES_H_
+
+#ifdef TABLE_FULL_SCALE
```

Comment:
will be better to use TABLE_MINI_HERO_SCALE

Reply:
TABLE_BABY_HERO_SCALE i believe it is in the range of 80K to 320K can we at least have the default  limits increased to that
![image](https://github.com/user-attachments/assets/71e84cc8-d808-438a-8435-952a7faf9182)

Reply:
Use TABLE_HERO_SCALE/TABLE_BABY_HERO_SCALE for hero/baby-hero respectively.